packages/atlas/src/components/_video/VideoThumbnail/VideoThumbnail.tsx

```diff
@@ -34,7 +34,7 @@ type PlaylistVideoThumbnailProps = {
   slots?: Pick<SlotsObject, 'bottomLeft' | 'topLeft' | 'center'>
 }
 type RegularVideoThumbnailProps = {
-  type: 'video'
+  type: 'thumbnail'
```

Comment:
Is this type even passed into the `useGetAssetUrl`? Could you point me to the file?

Reply:
https://github.com/Joystream/atlas/blob/c99fb287806b406bdb57a855c3d082c713de47ac/packages/atlas/src/hooks/useGetAssetUrl.ts#L14

https://github.com/Joystream/atlas/blob/c99fb287806b406bdb57a855c3d082c713de47ac/packages/atlas/src/providers/uploads/uploads.types.ts#L5